python/paddle/v2/framework/layers.py

```diff
@@ -204,6 +218,47 @@ def square_error_cost(input, label, **kwargs):
     return square_out
 
 
+def conv1d(input,
```

Comment:
Maybe `sequence_conv` is a better name?

Reply:
Rename `conv1d` to `sequence_conv` after discuss with @Canpio offline, since this function's behavior is different with `conv2d`, `conv3d`, fixed.